applications/crossbar/priv/couchdb/account/lists.json

```diff
@@ -3,7 +3,14 @@
     ,"language": "javascript"
     ,"views": {
         "crossbar_listing": {
-            "map": "function(doc) { if (doc.pvt_type == 'list' && !doc.pvt_deleted) emit(doc._id, {'id': doc._id, 'name': doc.name, 'entries': doc.entries}) };"
+            "map": "function(doc) {if (doc.pvt_deleted) return; doc.id = doc._id; if (doc.pvt_type == 'list_entry') emit(doc.list_id, {id: doc._id, pattern: doc.pattern, displayname: doc.displayname, firstname: doc.firstname, lastname: doc.lastname}); if (doc.pvt_type == 'list') emit(doc._id, doc);}",
```

Comment:
@jamesaimonetti 
 I'm sorry I didn't get it: should I

1) rename `crossbar_listing` to `crossbar_listing_v1` and `crossbar_listing_v2` to `crossbar_listing`
or
2) add `crossbar_listging_v1` identical to current `crossbar_listing`
?